views/outcome.erb

```diff
@@ -7,7 +7,25 @@
   <title>Rock Paper Scissors</title>
 </head>
 <body>
-  <h1><%= @player_name %> chose <%= @player_turn %></h1>
-  <h1>The computer chose <%= @computer_turn %></h1>
+  <p><%= @player_name %> chose <%= @player_turn %></p>
+  <p>The computer chose <%= @computer_turn %></p>
+
+  <% if @player_turn == "Rock" && @computer_turn == "Scissors" %>
```

Comment:
You could save yourself a load of lines of code if you had either the user/computer winning as an else statement